mut/stage.py

```diff
@@ -239,14 +239,16 @@ def __init__(self, verbose: bool, deployed_url_prefix: str) -> None:
 
         self.cache_control = CacheControl([])
 
-    def delete(self, objects: List[str], tag: str = 'D') -> None:
-        """Request deletion of a list of objects."""
-        self.commands_delete.extend((tag, x) for x in objects)
+    def delete(self, keys: List[str], flag: str = 'D') -> None:
+        """Request deletion of a list of keys."""
+        for key in keys:
+            self.commands_delete.append((flag, key))
+            self.full_deploy_urls.append((flag, self.deployed_url_prefix + '/' + key))
```

Comment:
Could we rename this to e.g. `full_modified_urls`?

Reply:
Done!